### PR TITLE
Studio lectures

### PR DESCRIPTION
**Pull request by JUtupe**

### CODE REVIEW

File: src/components/form/EventForm/Theme/EventThemeForm.tsx [hunk 1]
```diff
@@ -116,9 +120,9 @@ export const EventThemeForm: React.FC<EventThemeFormProps> = ({event, onEventCha
                 </StudioSection>
 
                 <div className={formStyles.submit}>
-                    <PrimaryButton type={"submit"}>
+                    <Button type={"submit"}>
```
Comment (DiD3n) [inline src/components/form/EventForm/Theme/EventThemeForm.tsx]: no need for `{curly}` bois

File: src/components/ui/Button/Button.tsx [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
+        onClick,
+        size = 'medium'
+    }
+) => {
     return (
         <button
             type={type}
             className={
                 cs(styles.button, {
+                    [styles.secondary]: secondary,
```
Comment (DiD3n) [inline src/components/ui/Button/Button.tsx]: uhhh, i would like to have primary as a prop

File: src/views/studio/EventCreate/EventCreateForm.tsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import React, {useState} from "react";
-import styles from "./EventCreateForm.module.scss";
+import studioFormStyles from "./../../../common/styles/StudioFormStyles.module.scss";
```
Comment (DiD3n) [inline src/views/studio/EventCreate/EventCreateForm.tsx:2]: name is too complicated 4me

Comment (JUtupe) [reply src/views/studio/EventCreate/EventCreateForm.tsx:2]: not 4 me hehe

File: src/components/ui/Button/Button.tsx (hunk 2 above)
Comment (JUtupe) [reply src/components/ui/Button/Button.tsx]: done

File: src/components/form/EventForm/Theme/EventThemeForm.tsx (hunk 1 above)
Comment (JUtupe) [reply src/components/form/EventForm/Theme/EventThemeForm.tsx]: I know but I want to keep them for consistency 😄

**Pull request merged**